src/OpenApi/sample/Transformers/OperationTransformers.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.OpenApi;
+using Microsoft.OpenApi.Models;
+using Microsoft.OpenApi.Any;
+
+public static class OperationTransformers
+{
+    public static OpenApiOptions AddHeader(this OpenApiOptions options, string headerName, string defaultValue)
+    {
+        return options.UseOperationTransformer((operation, context, cancellationToken) =>
+        {
+            operation.Parameters.Add(new OpenApiParameter
+            {
+                Name = headerName,
+                In = ParameterLocation.Header,
+                Schema = new OpenApiSchema
+                {
+                    Type = "string",
```

Comment:
Are there constants that can be used by people for these? Having to write out literals for these all the time feels like a recipe for typos when people start documenting things manually.

Reply:
For primitives like these, the `Microsoft.OpenApi` package provides [an API to generate the entire schema object](https://github.com/microsoft/OpenAPI.NET/blob/f10700ec571c56ef0959ae58de8e607171a67aff/src/Microsoft.OpenApi/Extensions/OpenApiTypeMapper.cs#L73) but doesn't provide constants for each schema type, AFAIK.

I can update this sample to take advantage of the API above.